client/src/main/java/io/pravega/client/stream/impl/Controller.java

```diff
@@ -219,5 +220,18 @@
      * @return Pravega node URI.
      */
     CompletableFuture<PravegaNodeUri> getEndpointForSegment(final String qualifiedSegmentName);
+    
+    /**
+     * Returns the total number of bytes from the provided positions until the stream.
+     * TODO: Make this API asynchronous.
```

Comment:
You point to the issue somewhere else, but it might be worth repeating the pointer here to avoid having this dangling TODO.